Cargo.toml

```diff
@@ -36,8 +36,6 @@ munsell = []
 argmin = "0.10.0"
 argmin-math = "0.4.0"
 clap = { version = "4.5.18", features = ["derive"] }
-getrandom = { version = "0.2", features = ["js"] }
```

Comment:
This can be seen as a follow up to #8. `getrandom` was completely unused. I could not find any point in the git history where it ever was used actually :see_no_evil: :sweat_smile: